api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1351,17 +1312,14 @@ public void purgeObs_shouldDeleteTheGivenObsFromTheDatabase() {
 		
 		
 		executeDataSet(COMPLEX_OBS_XML);
+		File complexObsDir = OpenmrsUtil.getDirectoryInApplicationDataDirectory(adminService
+			.getGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_COMPLEX_OBS_DIR));
+		File createdFile = createImage(complexObsDir, "openmrs_logo_small.gif");
+		
 		Obs complexObs = obsService.getComplexObs(44, ComplexObsHandler.RAW_VIEW);
```

Comment:
getCompexObs failed due to missing file with a new implementation. It appears that the old implementation didn't complain that the file didn't exist. I'm confused whether we should fail if the file is missing or silently ignore.

Reply:
@dkayiwa @ibacher do you have any opinion about that?

Reply:
In order to reduce the possibility of unintended side effects of introducing a new storage service, i would preserve the existing behaviour. Then if we need to change it to fail if the file is missing, i would do that as a separate ticket and pull request. 😊

Reply:
Changed to old behaviour in ImageHandler. Other handlers did fail if file was missing so we may want to fix that indeed in a separate issue.